model-mapper/src/Main.java

```diff
@@ -0,0 +1,58 @@
+import com.baeldung.model.User;
+import com.baeldung.model.UserDTO;
+import com.baeldung.model.UserList;
+import com.baeldung.util.MapperUtil;
+import com.baeldung.util.UserPropertyMap;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.TypeToken;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author sasam0320
+ * @date 4/18/2020
+ */
+
+public class Main {
+
+    public static void main(String[] args) {
```

Comment:
we don't use printlns or main methods. usually, we create unit tests or just methods in general when necessary, maybe we can do the same here.